tests/scripts/run_performance.sh

```diff
@@ -57,6 +57,8 @@ run_perf_models() {
 run_device_perf_models() {
     local pipeline_type=$1
 
+    tests/scripts/run_profiler_regressions.sh PROFILER
```

Comment:
Mind just adding a quick comment explaining why this is here (ie. clock cycles reset, need BM etc.)

I see you pass device perf regressions.

Then I'll approve

Reply:
Sounds good will do

Reply:
Done